downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -60,6 +60,20 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 #### Language Server
 
+- Introduced the `Extract to transform function` code action to extract a value expression into a transform function.
+- Introduced the `Surround with lock` code action to wrap an isolated variable access with a `lock` statement.
+- Introduced the `Add private qualifier` code action to set the visibility qualifier of a field in an isolated class to `private`.
+- Introduced the `Make variable immutable` code action to add `final` and/or `readonly` as needed to make the field immutable.
+
+#### Ballerina Shell
+
+- Added support for invoking actions directly from the shell prompt, as shown with the following examples.
```

Comment:
```suggestion
- Added support for invoking actions directly from the shell prompt, as shown in the following examples.

```

Reply:
You're correct. However, renaming the code action would introduce inconsistency with existing ones, like `Surround with do/on-fail`.